site/docs/operations/docker.md

```diff
@@ -278,6 +278,20 @@ To see the default content of the files run the following command:
 docker run hazelcast/hazelcast-jet cat /opt/hazelcast-jet/config/hazelcast-jet.yaml
 ```
 
+## Use Built-in Modules
```

Comment:
https://jet-start.sh/docs/operations/installation is it worth expanding this page as well? right now it doesn't talk so much about how to use built in modules.